<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin</class>
 <widget class="QDialog" name="Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>admin</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QComboBox" name="comboBox">
         <item>
          <property name="text">
           <string>查找全部</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>学员队</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QTableView" name="tableView"/>
       </item>
       <item row="1" column="3">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>查  找</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_del">
           <property name="text">
            <string>删  除</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_add">
           <property name="text">
            <string> 添  加</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_sort">
           <property name="text">
            <string>排  序</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButton_ok">
         <property name="text">
          <string>保存修改</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>245</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="pushButton_cal">
         <property name="text">
          <string>取消修改</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
